llmstudio/engine/providers/provider.py

```diff
@@ -253,7 +264,9 @@ def join_chunks(self, chunks, request):
                     part = chunk.get("arguments", "")
                     if part:
                         function_call_arguments += part
-                elif isinstance(request, OpenAIRequest):
+                elif isinstance(request, OpenAIRequest) or isinstance(
```

Comment:
do we need this condition? we're already with a request of instances OpenAIRequest or VertexAIRequest as defined on line 251